engines/director/castmember/castmember.h

```diff
@@ -86,6 +86,11 @@ class CastMember : public Object<CastMember> {
 
 	virtual Common::String formatInfo() { return Common::String(); };
 
+	virtual Common::Rect getBbox();
+	virtual Common::Rect getBbox(int16 width, int16 height);
```

Comment:
Same logic here, please describe it in a doxygen comment

Reply:
Done